src/software/jetson_nano/services/motor.h

```diff
@@ -82,12 +83,46 @@ class MotorService
      */
     void runOpenLoopCalibrationRoutine(uint8_t motor, size_t num_samples);
 
+    /**
+     * Reset the motor board by toggling the reset GPIO appropriately. Effectively stops
+     * the motors from moving.
+     */
+    void resetMotorBoard();
+
+    /**
+     * Clears previous faults, configures the motor and checks encoder connections.
+     */
+    void setup();
+
    private:
     /**
-     * Checks for faults, clears them and sets up motors.
-     *
+     * Holds motor fault information for a particular motor and whether any fault has
+     * caused the motor to be disabled.
      */
-    void setUpMotors();
+    struct MotorFaultIndicator
+    {
+        bool drive_enabled;
+        std::unordered_set<TbotsProto::MotorFault> motor_faults;
+
```

Comment:
Is there a reason why we can't use the existing `TbotsProto::DriveUnit` instead of this?